### ISSUE TITLE
Terminal command failed audio cue never played when screen reader mode is enabled

### ISSUE THREAD

**Issue by joyceerhl** [microsoft/vscode#205735]
Testing https://github.com/microsoft/vscode/issues/205384

1. Have screen reader mode enabled
2. Enable terminal command failed audio cue with the setting below
3. Run a command that fails like `asdf`
4. :bug: no sound is played
```json
"accessibility.signals.terminalCommandFailed": {
    "sound": "on"
}
```

**Comment by meganrogge** [comment]
@joyceerhl this works for me on mac. Is shell integration working for you in this case? That's what we rely upon here.

**Comment by joyceerhl** [comment]
I do get the shell integration decoration in this case

**Comment by meganrogge** [comment]
any errors in the dev tools console? 

https://github.com/microsoft/vscode/assets/29464607/20f21c6b-29dd-4a9a-8d1e-23c59358b20f

**Comment by joyceerhl** [comment]
Oh, I should mention I also had Narrator running, and whatever Narrator was saying might have played over the audio cue. If I just enable screen reader mode but don't have Narrator running, I do get the audio cue.

**Comment by meganrogge** [comment]
Could you pls try with NVDA? Sounds like an issue with Narrator, which is not one of the screen readers we recommend using.

**Comment by joyceerhl** [comment]
👍 works with NVDA
